the-9-exercises/src/components/List.jsx

```diff
@@ -0,0 +1,9 @@
+const List = (props) => {
+  return (
+    <div>
+      <li>{props.element}</li>
+    </div>
+  );
+};
+
+export default List;
```

Comment:
This component is called List but it does not really return a list instead it returns a list item (`<li></li>`) to avoid confusions the function name should explain what it's doing.